semantic_router/index/base.py

```diff
@@ -26,6 +26,7 @@ def add(
         embeddings: List[List[float]],
         routes: List[str],
         utterances: List[Any],
+        function_schemas: List[Dict[str, Any]] = None,  # type: ignore
```

Comment:
Should be avoiding use of `# type: ignore` unless required, here we should do:

```suggestion
        function_schemas: List[Dict[str, Any]] | None = None,
```